AndelaOSP/andela-societies-frontend
# [Feature #155907023] Display profile picture and username

**Description**
When a user logs in they should see an action button with their profile pic. When clicked a dropdown with their name should be revealed

**How to test**
A logged in user should be able to see their profile pic and username as described above

<img width="234" alt="screen shot 2018-03-21 at 17 04 16" src="https://user-images.githubusercontent.com/13675851/37714262-26dbcd58-2d2a-11e8-8d7e-36a905d9293d.png">
